ansible-ipi-install/roles/node-prep/vars/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# vars file for node-prep
+package_list: ['firewalld','libvirt', 'qemu-kvm', 'mkisofs', 'python3-devel', 'jq', 'ipmitool']
+virtpool_packages: ['python3-libvirt','python3-lxml']
+nmcli_packages: ['NetworkManager-libnm','nm-connection-editor','libsemanage-python3','policycoreutils-python3']
+ipmi_power_packages: ['https://trunk.rdoproject.org/rhel8-master/deps/latest/Packages/python3-crypto-2.6.1-18.el8ost.x86_64.rpm','https://trunk.rdoproject.org/rhel8-master/deps/latest/Packages/python3-pyghmi-1.0.22-2.el8ost.noarch.rpm']
```

Comment:
can't we gather those packages from an existing channel?

Reply:
@karmab @sreichar @e-minguez  - From what I gathered, the ipmi_power_packages I want to install reside in the OpenStack 15 repository. I didn't know of another way to handle this so I'm really open to suggestions. I don't think we want to use pip? I also don't think we want to force people to enable an OpenStack subscription to get the packages.